edx_proctoring/callbacks.py

```diff
@@ -43,11 +42,15 @@ def start_exam_callback(request, attempt_code):  # pylint: disable=unused-argume
         mark_exam_attempt_as_ready(attempt['proctored_exam']['id'], attempt['user']['id'])
 
     log.info("Exam %r has been marked as ready", attempt['proctored_exam']['id'])
-    template = loader.get_template('proctored_exam/proctoring_launch_callback.html')
-
-    return HttpResponse(
-        template.render({
-            'platform_name': settings.PLATFORM_NAME,
-            'link_urls': settings.PROCTORING_SETTINGS.get('LINK_URLS', {})
-        })
-    )
+
+    course_id = attempt['proctored_exam']['course_id']
```

Comment:
I can't leave this comment on the actual line since it hasn't been modified, but on line 23, there's a comment: `NOTE: This returns HTML as it will be displayed in an embedded browser`. It is no longer correct, since we're now redirecting to the exam URL instead of returning HTML. Could you either correct the comment or remove it, please?